pkg/kubelet/dockertools/manager.go

```diff
@@ -1230,7 +1230,7 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 }
 
 // createPodInfraContainer starts the pod infra container for a pod. Returns the docker container ID of the newly created container.
-func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubeletTypes.DockerID, error) {
+func (dm *DockerManager) createPodInfraContainer(pod *api.Pod, pullSecrets []api.Secret) (kubeletTypes.DockerID, error) {
```

Comment:
> The infrastructure container shouldn't need the pull secrets, it can just pass nil to PullImage. Right?

It will **never** need them for any possible infra container (I'm pretty sure this overrideable) or it doesn't need them for the kube one?  If it's the latter, its probably correct to continue passing these.

Nil is acceptable because `range` and `len` work.